Sources/DatadogCrashReporting/DDCrashReportingPlugin.swift

```diff
@@ -0,0 +1,35 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+import CrashReporter
+
+public class DDCrashReportingPlugin {
+    private static var sharedPLCrashReporter: PLCrashReporter?
+
+    public init?() {
+        DDCrashReportingPlugin.sharedPLCrashReporter = PLCrashReporter(
+            configuration: PLCrashReporterConfig(
+                signalHandlerType: .BSD,
+                symbolicationStrategy: .all
+            )
+        )
+    }
+
+    // TODO: RUMM-956 Revamp this by shaping the final API
+    public func testIfItWorks() {
```

Comment:
![So](https://media.giphy.com/media/xT5LMJigexzc97PdeM/giphy.gif)

Reply:
Yeah, this works 👌. But struggling some Carthage issues, so this PR is still draft 🙂.